aya/src/maps/mod.rs

```diff
@@ -71,8 +75,11 @@ pub mod stack;
 pub mod stack_trace;
 
 pub use array::{Array, PerCpuArray, ProgramArray};
+pub use bloom_filter::BloomFilter;
 pub use hash_map::{HashMap, PerCpuHashMap};
-pub use map_lock::*;
+pub use lpm_trie::LpmTrie;
+#[cfg(any(feature = "async", doc))]
```

Comment:
See: #404 
This should ideally be:
```
#[cfg(feature = "async")]
#[cfg_attr(docsrs, doc(cfg(feature = "async")))]
```

`doc` isn't required as we build docs with all features enabled.
And this will render nicely in the docs...

I'm not sure if not re-exporting AsyncPerfArray was an intentional decision or not, but overall I'm +1 on this.

Reply:
Sweet makes sense I update the feature macro here, the map enum, and in maps/perf/mod.rs

> I'm not sure if not re-exporting AsyncPerfArray was an intentional decision or not, but overall I'm +1 on this.

Basically in maps/mod.rs I wanted  to ensure we only use AsyncPerfArray/ add it to the `Maps` enum if the feature is toggled, not sure if this is what you meant :)

Reply:
Done!